needle.pm

```diff
@@ -311,9 +311,13 @@ sub wanted_ {
 sub default_needles_dir { "$bmwqemu::vars{PRODUCTDIR}/needles" }
 
 sub init {
-    $needles_dir = ($bmwqemu::vars{NEEDLES_DIR} // default_needles_dir);
-    $needles_dir = File::Spec->catdir($bmwqemu::vars{CASEDIR}, $needles_dir) unless -d $needles_dir;
-    die "needles_dir not found: $needles_dir (check vars.json?)" unless -d $needles_dir;
+    my $default_needles_dir = ($bmwqemu::vars{NEEDLES_DIR} // default_needles_dir);
+    $needles_dir = $default_needles_dir;
```

Comment:
I would prefer to use a different variable name than `$default_needles_dir` because you also read the `NEEDLES_DIR` variable

Reply:
Do you have any suggestion? this parameter only be used in `die` message.

Reply:
Ok, sorry that I didn't provide a suggestion. How about `$needles_dir_from_settings` or `$init_needles_dir`?

Reply:
Thanks for your ideas. I prefer `$init_needles_dir`.